examples/racetrack/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Racetrack</title>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+
+    <script src="racetrack.js" defer></script>
+  </head>
+  <body>
+    <img src="assets/car.png" id="car" class="car">
```

Comment:
Do we need the `class` car here as well? It's not being used in the CSS, but I understand if it's intended as instructional.

Reply:
Ah yea, should remove it. I just added it during the demo when discussing the difference between classes and IDs

Reply:
Good catch